blivet/devicelibs/dasd.py

```diff
@@ -114,6 +114,10 @@ class dasd_info(ctypes.Structure):
 
 def is_ldl_dasd(device):
     """Determine whether or not a DASD is LDL formatted."""
+    statusfile = "/sys/block/%s/device/status" % (device,)
+    if not os.path.isfile(statusfile):
+        return False
```

Comment:
Could you achieve the same result by checking that device type is "dasd" before calling `is_ldl_dasd`? (I assume it's being called from anaconda)